sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageStreamSource.java

```diff
@@ -335,13 +341,15 @@ public synchronized void close() {
                   + "the left of the split fraction {}.",
               source.readStream.getName(),
               fraction);
+          splitPossible = false;
```

Comment:
Nit: I don't think this is correct, or required. Once we've reached this point, we know that the splitReadStream call above has in fact succeeded.

Reply:
Removed.